src/virt_ctlr_combined.cpp

```diff
@@ -413,3 +420,34 @@ bool virt_ctlr_combined::mac_belongs(const std::string& mac) const
     return mac != "" && (mac == left_mac || mac == right_mac);
 }
 
+bool virt_ctlr_combined::set_player_led(int index, bool on)
+{
+    if (index > 3)
+        return false;
+
+    libevdev_uinput_write_event(uidev, EV_LED, 0, on ? LIBEVDEV_LED_ON : LIBEVDEV_LED_OFF);
```

Comment:
```suggestion
    libevdev_uinput_write_event(uidev, EV_LED, index, on);
```

I've tested it, and the write_event doesn't work with these constants. Seems like it sets the leds on/off based on boolean values, so in both cases (LIBEVDEV_LED_OFF == 3, LIBEVDEV_LED_ON == 4), the led is turned on. Using the boolean argument directly works.